src/commands/preview/upload.rs

```diff
@@ -78,7 +78,7 @@ pub fn upload(
                 "You haven't run `wrangler config`. Running preview without authentication",
             );
             message::help(
-                "Run `wrangler config` or set $CF_API_KEY and $CF_EMAIL to configure your user.",
+                "Run `wrangler config` or set $CF_EMAIL and $CF_API_KEY/$CF_API_TOKEN to configure your user.",
```

Comment:
I find this a bit confusing - can we just promote `$CF_API_TOKEN` here?

Reply:
yes!

Reply:
```suggestion
                "Run `wrangler config` or set either of $CF_API_TOKEN or ($CF_EMAIL, $CF_API_KEY) to configure your user.",
```